### ISSUE TITLE
Create the "Contact Us" Page 

### ISSUE THREAD

**Issue by arshadpatel** [arshadpatel/codeX100#9]
Build a "Contact Us" page where users can reach out with questions or feedback. The form should collect the user’s name, email, and message, and include form validation.

**Tasks:**

- Create the contact form layout using HTML and CSS.
- Add client-side validation for the email and message fields.
- Design a submit button that sends the form (without backend implementation for now).

**Skills:** HTML, CSS, JavaScript

**Comment by harshit1142** [comment]
@arshadpatel Can you assign me ?

**Comment by arshadpatel** [comment]
@harshit1142 Go ahead you can start working on it

**Comment by harshit1142** [comment]
@arshadpatel there is a certain procedure.

**Comment by arshadpatel** [comment]
Yes I am aware of it as I am also a open-source contributor... But  I'm not assigning issues to individuals at this time. Contributors can jump in and start working on the issues directly.

The first person to submit a valid pull request will have their contribution merged. If multiple people submit pull requests for the same issue, I will review all submissions, and if the code is exceptional, I will merge it regardless of who submitted it first.

Feel free to start working on the issue, and I look forward to seeing your contributions!

**Comment by uzefaqureshi06** [comment]
can you assign this to me I will work on it

**Comment by arshadpatel** [comment]
@uzefaqureshi06 You can jump in and start working on the issue directly. Feel free to start working on the issue, and I look forward to seeing your contributions!

**Comment by uzefaqureshi06** [comment]
hey can you plz merge it

**Comment by arshadpatel** [comment]
@uzefaqureshi06 Let me review it... I will let you know if additional enhancement required...

**Comment by arshadpatel** [comment]
@uzefaqureshi06 Please open the pull request so that I can merge it...

**Comment by uzefaqureshi06** [comment]
@arshadpatel i had raised the PR can you review it and merge it

**Comment by arshadpatel** [comment]
Please mention the reference so that it will be easier for me to check directly from here
